### PR TITLE
DLRM model checksum + corrected pre download clean

### PR DESCRIPTION
**Pull request by anandhu-eng**
This PR makes progress to issue https://github.com/mlcommons/cm4mlops/issues/167.

Other modifications:
1. run.sh is modified as there is no `CM_PRE_DOWNLOAD_CLEAN` env variable by default and we need to delete already existing incomplete download file.
2. default value for pre clean is set to `true`.  The incomplete downloads will not be cleared when user explicitly sets `CM_PRE_DOWNLOAD_CLEAN` to `false`

### CODE REVIEW

File: script/download-file/run.sh [hunk 1]
```diff
@@ -35,10 +35,8 @@ fi
 
 if [[ ${require_download} == "1" ]]; then
   echo ""
-  if [ -e "${CM_PRE_DOWNLOAD_CLEAN}" ]; then
-    echo ${CM_PRE_DOWNLOAD_CLEAN_CMD}
-    ${CM_PRE_DOWNLOAD_CLEAN_CMD}
-  fi
+  echo ${CM_PRE_DOWNLOAD_CLEAN_CMD}
```
Comment (arjunsuresh) [inline script/download-file/run.sh:38]: Why this change is needed?

File: script/download-file/run.sh [hunk 2]
```diff
@@ -26,6 +26,9 @@ elif [ -e "${CM_DOWNLOAD_DOWNLOADED_PATH}" ]; then
        # checksum not supposed to fail for locally given file
        if [[ "${CM_DOWNLOAD_LOCAL_FILE_PATH}" != "" ]]; then
           exit 1
+       else
+          echo "Checksum mismatch. Deleting through command: ${CM_PRE_DOWNLOAD_CLEAN_CMD}"
```
Comment (arjunsuresh) [inline script/download-file/run.sh:30]: `CM_PRE_DOWNLOAD_CLEAN=1`

Is a cleaner code right?

Comment (anandhu-eng) [reply script/download-file/run.sh]: Hi  @arjunsuresh , its assigned as `true` in commit [b40ea46](https://github.com/GATEOverflow/cm4mlops/pull/125/commits/b40ea46bbe1b24c250eb36af58a2e03fd5018a28) to make it uniform since in `customize.py`, it expects true or false.

File: script/download-file/run.sh [hunk 3]
```diff
@@ -35,7 +37,7 @@ fi
 
 if [[ ${require_download} == "1" ]]; then
   echo ""
-  if [ -e "${CM_PRE_DOWNLOAD_CLEAN}" ]; then
+  if [ "${CM_PRE_DOWNLOAD_CLEAN}" != "" ]; then
```
Comment (arjunsuresh) [inline script/download-file/run.sh:40]: We should check for not equal to "False" here

**Review by arjunsuresh — APPROVED** [review]
